README.rst

```diff
@@ -98,45 +98,47 @@ Documentation (WIP)
 
 The `full documentation <https://serenata_toolbox.readthedocs.io>`_ is still a work in progress. If you wanna give us a hand you will need `Sphinx <http://www.sphinx-doc.org/>`_:
 
-::
+.. code-block:: bash
 
   $ cd docs
   $ make clean;make rst;rm source/modules.rst;make html
 
 Contributing
 ------------
 
-Within your `virtualenv <https://virtualenv.pypa.io/en/stable/>`_:
+Firstly, you should create a development environment with `virtualenv <https://virtualenv.pypa.io/en/stable/>`_ to isolate your development.
```

Comment:
Didn't you like [my suggestion to update the reference link to Python official docs](https://github.com/okfn-brasil/serenata-toolbox/pull/194#discussion_r213675625) (as well as replacing `virtualenv` by `venv` as it's called in Python 3.6)?

Reply:
Updated.